src/components/client/hero/hero.tsx

```diff
@@ -148,6 +149,7 @@ export const Hero = () => {
           alt="slider"
         />
       </div>
+      <Slider />
```

Comment:
1. this should not be in hero, add in page
2. slider should be made reusable
props: title, subTitle, array with product 
3. example arr product arr: IProduct.d.ts

Reply:
Done